### PR TITLE
Release a new version description

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,24 @@ and/or with `docker-compose`:
 ```
 docker-compose build && docker-compose run --rm app go test ./...
 ```
+
+### Release a new version
+
+1. Release a new versions of bitrise-tools (stepman, envman) if there are changes
+1. Release a new versions of default plugins if there are changes
```
Comment (viktorbenei) [inline README.md]: list those, or the file where to find the list

File: README.md [hunk 2]
```diff
@@ -95,3 +95,24 @@ and/or with `docker-compose`:
 ```
 docker-compose build && docker-compose run --rm app go test ./...
 ```
+
+### Release a new version
+
+1. Update go dependencies (`bitrise run godeps-update`)
+1. PR & merge these changes to the `master` branch
```
Comment (trapacska) [inline README.md]: 2. ? 😊

File: README.md [hunk 3]
```diff
@@ -95,3 +95,24 @@ and/or with `docker-compose`:
...
+1. Release a new versions of bitrise-tools (stepman, envman) if there are changes, you can find the dependent tools in `./bitrise/setup.go -> minEnvmanVersion, minStepmanVersion`
```
Comment (trapacska) [inline README.md]: 3. ? 😊

File: README.md (hunk 2 above)
Comment (viktorbenei) [inline README.md]: nop, markdown allows you to list with `1. 1. 1.` and will generate a proper list like this

1. first
1. second
1. third

**Pull request merged**